pkg/controller/rsm/pod_role_event_handler.go

```diff
@@ -135,14 +135,19 @@ func handleRoleChangedEvent(cli client.Client, reqCtx intctrlutil.RequestCtx, re
 			}
 		}
 
-		name, _ := intctrlutil.GetParentNameAndOrdinal(pod)
-		rsm := &workloads.InstanceSet{}
-		if err := cli.Get(reqCtx.Ctx, types.NamespacedName{Namespace: pod.Namespace, Name: name}, rsm); err != nil {
+		var name string
+		if pod.Labels != nil {
```

Comment:
Do we need to consider compatibility with 0.8？

Reply:
will align all labels to 0.9 when doing upgrade